fairscale/utils/testing.py

```diff
@@ -303,7 +304,14 @@ def replacement(*args: Any, **kwargs: Any) -> None:
 
             error_queue = multiprocessing.get_context("spawn").SimpleQueue()
             if "OMPI_COMM_WORLD_RANK" in os.environ:
+                # TODO (Min): this global used to be assigned every time this file is imported.
+                #     I changed it to be assigned on first use. Should be the same, but I am not
+                #     sure this is used or is correct since different processes would have different
+                #     file names to init_process_group below. By initing, here, we don't leave
+                #     a temp file behind on importing time.
```

Comment:
```suggestion
                # TODO (Min): this global used to be assigned every time this file is imported.
                # I changed it to be assigned on first use. Should be the same, but I am not
                # sure this is used or is correct since different processes would have different
                # file names to init_process_group below. By initing, here, we don't leave
                # a temp file behind on importing time.
```